js-tests/build/specs.js

```diff
@@ -281,39 +281,39 @@ describe( "MCE View Constructor", function() {
 	} );
 
 	it( 'parses simple shortcode', function() {
-		var shortcode = MceViewConstructor.parseShortcodeString( '[test_shortcode attr="test value"]')
+		var shortcode = MceViewConstructor.parseShortcodeString( '[test_shortcode attr="test value"]');
```

Comment:
Looks like all the highlighted lines here are missing a space before the closing `)`?